server/rpc_test.go

```diff
@@ -532,3 +534,59 @@ func defaultTicket(t *testing.T) *net.Ticket {
 		RecipientRandHash: pm.RandBytes(123),
 	}
 }
+
+func StubBroadcastSessionsManager() *BroadcastSessionsManager {
+	sess1 := &BroadcastSession{
+		Broadcaster: StubBroadcaster2(),
+		ManifestID:  core.RandomManifestID(),
+	}
+	sess2 := &BroadcastSession{
+		Broadcaster: StubBroadcaster2(),
+		ManifestID:  core.RandomManifestID(),
+	}
+	return &BroadcastSessionsManager{broadcastSessions: []*BroadcastSession{sess1, sess2}, sessLock: &sync.Mutex{}}
+}
+
+func TestSelectFromList_Successt(t *testing.T) {
+	bsm := StubBroadcastSessionsManager()
+	expectedSess := bsm.broadcastSessions[0]
+	sess := bsm.selectFromList()
```

Comment:
Also test the empty-list case, and check that the list lengths are as expected before-and-after the function call.